builder/Dockerfile

```diff
@@ -148,8 +194,8 @@ FROM ${base} as kat-server
 COPY --from=artifacts /buildroot/bin/kat-server /usr/bin/kat-server
 
 WORKDIR /work
-COPY server.crt server.crt
-COPY server.key server.key
+COPY builder/server.crt server.crt
+COPY builder/server.key server.key
```

Comment:
We run docker build from the top level so we need to reference these files inside the builder/ directory.